src/redux/watcher.js

```diff
@@ -97,6 +98,7 @@ export const cbeWatcher = () => (dispatch) => {
   // voting TODO MINT-93 use watchInit* and _watch
   VoteDAO.newPollWatch((index) => dispatch(handleNewPoll(index)))
   VoteDAO.newVoteWatch((index) => dispatch(handleNewVote(index)))
+  dispatch(watchInitRewards())
```

Comment:
this dispatch should be inside _watcher_ function and not _cbeWatcher_, because Rewards not only for CBE users